libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/OnNavigationReadyCallback.java

```diff
@@ -2,5 +2,5 @@
 
 public interface OnNavigationReadyCallback {
 
-  void onNavigationReady();
+  void onNavigationReady(boolean isRunning);
```

Comment:
Can we apply the boy-scout rule here and provide some javadoc for this class?  Main explanation will be `isRunning` will be true after you've initialized the `NavigationView` once and began navigation.  Indicates that `NavigationViewModel` is currently running and hasn't been destroyed from a configuration change.